Subscribed to Fixed Issue #19905 Newsletter label and checkbox not aligned proper in admin panel while any customer editing

GovindaSharma opened the pull request:
Fixed Issue #19905 Newsletter label and checkbox not aligned proper in admin panel while any customer editing

### Description (*)

 Fixed Issue #19905 Newsletter label and checkbox not aligned proper in admin panel while any customer editing

### Fixed Issues (if relevant)

1. magento/magento2 #19905:  Newsletter label and checkbox not aligned proper in admin panel while any customer editing

### Manual testing scenarios (*)

Step 1: Go to admin url
Step 2: login to admin panel
Step 3: click on Customers link from left side and under customers click on all customers
Step 4: You see all registered customer grid now you edit any one of them.
Step 5: Now click on newsletter tab and check Newsletter checkbox is not aligned properly

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

Commit:
Fixed Issue #19891 by adding checks of 'type_id' attribute

Fixed Issue #19891 by adding checks of 'type_id' attribute

Commit:
Fixed issue #19905 Newsletter checkbox alignment issue

Fixed issue #19905 Newsletter checkbox alignment issue

Commit:
Reverted The Chnages

Commit:
Fixed Issue #19905 .Issue fixed Newletter checkbox Alignment

Fixed Issue #19905 .Issue fixed Newletter checkbox Alignment

magento-engcom-team commented:
Hi @GovindaSharma. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

VladimirZaets commented:
Hi @GovindaSharma. Thanks for collaboration.  As I see it is an alignment issue, so what the reason to change controller file `app/code/Magento/Catalog/Controller/Adminhtml/Product/Attribute/Validate.php`?

Commit:
Reverted the suggested changes.

GovindaSharma commented:
Hi @VladimirZaets ,actually by mistake another branch code gets merged in new branch code ,as the previous code of controller which i had changed is for another pull request which is in pending.So now i reverted another branch code.

GovindaSharma commented:
@VladimirZaets .Please check and let me know further updates.
